sources/packages/web/src/services/http/ConfirmationOfEnrollmentApi.ts

```diff
@@ -42,18 +42,20 @@ export class ConfirmationOfEnrollmentApi extends HttpBaseClient {
 
   public async confirmCOE(
     locationId: number,
-    applicationId: number,
+    disbursementScheduleId: number,
   ): Promise<void> {
-    try {
-      await this.apiClient.patch(
-        `institution/location/${locationId}/confirmation-of-enrollment/application/${applicationId}/confirm`,
+    await this.apiClient
+      .patch(
```

Comment:
we have `patchCall()`
![image](https://user-images.githubusercontent.com/77353155/153086394-b9dea29d-b02c-429a-996d-e4a62b6bad25.png)